server/src/Config.hs

```diff
@@ -105,6 +107,11 @@ configParser environment = Config
   <*> storageBackend
   <*> switch (long "sync-log" <>
              help "Enable logging the duration of Sync operations.")
+  <*> option auto
+       (long "websocket-ping-interval" <>
+        metavar "WS-PING-INTERVAL" <>
+        value 30 <>
+        help "The interval of time between two pings to the WebSocket clients. It is instrumental in keeping connexions alive.")
```

Comment:
Somewhere this should mention that this value is specified in seconds. (Could just use `metavar "SECONDS"` - then it shows up as `--websocket-ping-interval SECONDS` in the help text)

Also there's a small typo: "connections" rather than "connexions".